packages/c-avatar/src/c-avatar.tsx

```diff
@@ -0,0 +1,215 @@
+          </g>
+        </svg>
+      </CBox>
+    )
+  },
+})
+
+const CAvatarInitials = defineComponent({
+  props: {
+    name: {
+      type: String,
+      default: "",
+    },
+    initials: {
+      type: String,
+      default: "",
+    },
+  },
+  setup(props) {
+    const styles = useStyles()
+    const containerStyles = useDynamicContainerStyles(props, styles)
+
+    const labelStyles = computed(() => ({
+      ...styles.value.label,
+      fontWeight: "medium",
+    }))
+
+    return () => (
+      <CBox __css={containerStyles.value}>
+        <chakra.div
+          role="img"
+          aria-label={props.name}
+          __css={labelStyles.value}
+        >
+          {props.initials
+            ? props.initials
+            : props.name && getInitials(props.name)}
+        </chakra.div>
+      </CBox>
+    )
+  },
+})
+
+export const avatarProps = {
+  as: {
+    type: [Object, String] as PropType<DOMElements>,
+    default: "span",
+  },
+  src: {
+    type: String,
+    default: "",
+  },
+  name: {
+    type: String,
+    default: "",
+  },
+  initials: {
+    type: String,
+    default: "",
+  },
+  ...vueThemingProps,
+}
+
+export type CAvatarProps = typeof avatarProps
+
+export const CAvatar: ComponentWithProps<CAvatarProps> = defineComponent({
+  props: {
+    ...avatarProps,
+  },
+  setup(props, { slots, attrs }) {
+    // Props handling
+    const mergedProps = computed(() => mergeProps({}, props, attrs))
+    const ownProps = computed(() => omitThemingProps(mergedProps.value))
+    const extractedAttrs = computed(() => extractImgAttrs(ownProps.value))
+
+    // State handling
+    const error = ref(false)
+    const pending = ref(true)
+
+    // Fetching custom icon
+    const validChildren = ref(getValidChildren(slots))
+    const customIcon = validChildren.value?.find((child, index) => {
+      if ((child.type as any).name === "CIcon") {
+        validChildren.value.splice(index, 1)
+        return true
+      }
+    })
+
+    // Handling styles
+    const avatarGroupStyles = useAvatarGroup()
+
+    const themingProps = computed<ThemingProps>(() =>
+      filterUndefined({
+        colorScheme: avatarGroupStyles?.value?.colorScheme || props.colorScheme,
+        variant: avatarGroupStyles?.value?.variant || props.variant,
+        size: avatarGroupStyles?.value?.size || props.size,
+        styleConfig: avatarGroupStyles?.value?.styleConfig || props.styleConfig,
+      })
+    )
+
+    const styles = useMultiStyleConfig("Avatar", themingProps)
+    const containerStyles = useDynamicContainerStyles(props, styles)
+
+    const imgStyles = computed(() => ({
+      objectFit: "cover",
+      ...containerStyles.value,
+    }))
+
+    StylesProvider(styles)
+
+    return () => (
+      <chakra.div __css={containerStyles.value} {...extractedAttrs.value.rest}>
```

Comment:
Let's add the chakra `__label` prop to the component so the developer can debug it easier
```suggestion
      <chakra.div __label="avatar" __css={containerStyles.value} {...extractedAttrs.value.rest}>
```